include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -193,6 +194,7 @@ typedef OrtStatus* OrtStatusPtr;
   _Success_(return == 0) _Check_return_ _Ret_maybenull_ OrtStatusPtr ORT_API_CALL NAME(__VA_ARGS__) NO_EXCEPTION
 
 #define ORT_CLASS_RELEASE(X) void(ORT_API_CALL * Release##X)(_Frees_ptr_opt_ Ort##X * input)
+#define ORT_CLASS_RELEASE2(X) void(ORT_API_CALL * Release##X)(_Frees_ptr_opt_ Ort##X##V2 * input)
```

Comment:
Why do we need a separate macro?

Reply:
Explained on other comment's replied.

The main reason is to make following four API's names consistent.
CreateTensorRTProviderOptions, UpdateTensorRTProviderOptions , GetTensorRTProviderOptions and ReleaseTensorRTProviderOptions